dubbo-spring-boot/dubbo-spring-boot-observability-starter/src/main/resources/META-INF/spring-configuration-metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+  "groups": [
+    {
+      "name": "dubbo.tracing",
+      "type": "org.apache.dubbo.spring.boot.observability.config.DubboTracingProperties",
+      "sourceType": "org.apache.dubbo.spring.boot.observability.config.DubboTracingProperties"
+    }
+  ],
+  "properties": [
+    {
+      "name": "dubbo.tracing.baggage",
```

Comment:
Is this related to baggage really or context propagation type? Cause how baggage is done is related to context propagation type really

Reply:
can i rename it to dubbo.tracing.propagation?

Reply:
This is how we do it in Boot - https://github.com/spring-projects/spring-boot/blob/main/spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/TracingProperties.java#L171 `management.tracing.propagation.type`

Reply:
> This is how we do it in Boot - https://github.com/spring-projects/spring-boot/blob/main/spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/TracingProperties.java#L171 `management.tracing.propagation.type`

OK